### DISCUSSION TITLE
use terminal colorscheme as default if not install plugin color scheme

### DISCUSSION THREAD

**Discussion by Phantasimay** [neovim/neovim#27995] [General]
### Problem

hi, i have little problem to use my terminal colorscheme as my nvim colorscheme after  feat(highlight): update default color scheme #26334  merge. can you guys help me how to use terminal colorscheme as nvim colorscheme. before it merge nvim magicaly detect my terminal colorscheme.
 

### Expected behavior

in default nvim config auto detect terminal colorscheme as nvim colorscheme

**Comment by Phantasimay** [comment]
TERM Alacritty
Cholorscheme TERM custom dracula
NVIM v0.10.0-dev-2674+ga62988842

Before merge:
![screenshot-23-03-24-20:08:44](https://github.com/neovim/neovim/assets/119695017/165690d4-a254-42fa-88a7-55b8ca0f5913)


After merge :
![screenshot-23-03-24-20:16:25](https://github.com/neovim/neovim/assets/119695017/416ff2f2-f122-48e2-bebf-6d7163b98711)


btw i very very love your work guys. thanks for up grade vim.

**Reply by justinmk** [reply]
the "Before merge" pic has the old vim colors. To get the old vim colors, use the "vim" theme:

    :colors vim

**Reply by clason** [reply]
`:set notermguicolors`

**Comment by Phantasimay** [comment]
> the "Before merge" pic has the old vim colors. To get the old vim colors, use the "vim" theme:
> 
> ```
> :colors vim
> ```
yes it work thanks
